srcpkgs/suricata/template

```diff
@@ -1,17 +1,22 @@
 # Template file for 'suricata'
 pkgname=suricata
 version=4.1.2
-revision=1
+revision=2
 build_style=gnu-configure
-configure_args="--disable-gccmarch-native"
+configure_args="--disable-gccmarch-native $(vopt_enable rust_support rust) $(vopt_enable luajit)"
 hostmakedepends="pkg-config"
-makedepends="libnet-devel libpcap-devel pcre-devel libyaml-devel libcap-ng-devel file-devel"
+makedepends="libnet-devel libpcap-devel pcre-devel libyaml-devel libcap-ng-devel liblz4-devel file-devel jansson-devel nss-devel"
 short_desc="Next Generation Intrusion Detection and Prevention Engine"
 maintainer="Andrea Brancaleoni <abc@pompel.me>"
 license="GPL-2"
 homepage="https://suricata-ids.org/"
 distfiles="https://www.openinfosecfoundation.org/download/$pkgname-$version.tar.gz"
 checksum=73575b041a50cc48a2a53f6503ab4d355166d7acbd4997cd04045f848f8bea96
+makedepends+=" $(vopt_if luajit LuaJIT-devel) $(vopt_if rust_support rust) $(vopt_if rust_support cargo)"
+build_options="luajit rust_support"
+build_options_default="luajit rust_support"
+desc_option_luajit="Enable lua jit support"
```

Comment:
The `luajit` option needs to be disabled for cross compilation as it is not available for the architectures that are currently cross compiled.

Reply:
fixed. unfortunately, suricata's configure script doesn't like --enable-luajit --disable-lua, so i had to resort to the vopt_if hack. i still enabled lua without jit.

Reply:
Not sure if this is a good idea. The packages should be as close as possible, switching from luajit which is like lua51 to lua53 if its cross compiling is something I would like to avoid.
Even then, if the package should be built with luajit by default, use luajit or disable it.
Having completely different dependencies based on how the package was build is confusing.

Reply:
agreed. i disable the luajit option for cross compiling. but i keep the lua option if some really needs this on a rpi whatever.